### PR TITLE
Add sprig

### PR DESCRIPTION
**Pull request by so0k**
To support advanced templating functions

### CODE REVIEW

File: ghac/template.go [hunk 1]
```diff
@@ -4,11 +4,14 @@ import (
...
 	"io"
 	"path"
+
+	"github.com/Masterminds/sprig"
 )
 
 // RenderTemplate renders Team using tplFile into target Writer
 func RenderTemplate(t *Team, tplFile string, wr io.Writer) error {
 	tpl, err := template.New(path.Base(tplFile)).
+		Funcs(sprig.FuncMap()).
```
Comment (tuannvm) [inline ghac/template.go]: So we no longer need the wrapper struct, right?

File: ghac/templates/team.tf.tpl [hunk 2]
```diff
@@ -1,9 +1,10 @@
 resource "github_team" "{{.Slug}}" {
   name        = "{{.Name}}"
   description = "{{.Description}}"
-  privacy     = "{{.Privacy}}"
+  privacy     = "{{ default "closed" .Privacy }}"
```
Comment (tuannvm) [inline ghac/templates/team.tf.tpl]: 💯

File: ghac/template.go (hunk 1 above)
Comment (so0k) [reply ghac/template.go]: see my comments about that on the PRs I've referenced

Comment (tuannvm) [reply ghac/template.go]: ok

**Pull request merged**